stan/math/fwd/meta/value_type.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef STAN_MATH_FWD_META_VALUE_TYPE_HPP
+#define STAN_MATH_FWD_META_VALUE_TYPE_HPP
+
+#include <stan/math/prim/meta/value_type.hpp>
+#include <stan/math/prim/meta/is_fvar.hpp>
+#include <stan/math/fwd/core/fvar.hpp>
+
+#include <type_traits>
+
+namespace stan {
+
+template <typename T>
+struct value_type<T, require_fvar_t<T>> {
+  using type = typename std::decay_t<T>::Scalar;
```

Comment:
So now a `value_type` of an `fvar<T>` is `T` if I'm reading this correctly. Is this different than it was before?

Reply:
I'm assuming not because then presumably lots of tests would break. Did we just not use `value_type` on `fvar` s?

I guess also what is this change for. Just so I understand

Reply:
> Did we just not use value_type on fvar s?

So it seems (or we did not notice we do not get back what we would expect).

Reply:
I think delete the value_type thing now that it's unused. Don't want to accidentally make a decision we don't need in these weird template type things.

Reply:
Can we you delete the value_type specialization for fvar since it isn't used now? That's all I'm waiting on here.